src/main/resources/db/migration/V20211001_21__HMAN-21-HearingService.sql

```diff
@@ -0,0 +1,153 @@
+CREATE TABLE public.case_hearing_request (
+                         auto_list_flag boolean not null,
+                         hearing_type varchar(40) not null,
+                         required_duration_in_minutes integer not null,
+                         hearing_priority_type varchar(60) not null,
+                         number_of_physical_attendees integer,
+                         hearing_in_welsh_flag boolean,
+                         private_hearing_required_flag boolean,
+                         lead_judge_contract_type varchar(70),
+                         first_date_time_of_hearing_must_be timestamp without time zone,
+                         hmcts_service_id varchar not null,
```

Comment:
```suggestion
                         hmcts_service_id varchar(4) not null,
```
Based on the pattern `^\w{4}$` we can set this to 4